packages/headless/src/auth/isomorphic-history.ts

```diff
@@ -0,0 +1,12 @@
+export interface IsomorphicHistory {
```

Comment:
Can we derive it from `History` ? 

Something like: 
```
type IsomorphicHistory = Pick<History, 'replaceState'>
```

Reply:
Great suggestion, adjusting 💯